astra/src/main/java/com/slack/astra/chunkrollover/DiskOrMessageCountBasedRolloverStrategy.java

```diff
@@ -44,20 +47,36 @@ public class DiskOrMessageCountBasedRolloverStrategy implements ChunkRollOverStr
 
   private final AtomicReference<FSDirectory> activeChunkDirectory = new AtomicReference<>();
   private final AtomicLong approximateDirectoryBytes = new AtomicLong(0);
+  private final AtomicBoolean maxTimePerChunksMinsReached = new AtomicBoolean(false);
+  private final Instant chunkStartTime;
 
   public static DiskOrMessageCountBasedRolloverStrategy fromConfig(
```

Comment:
We should probably (eventually?) rename this. Also I don't think the max messages rollover is helpful at all for the indexer today, so that probably could also (eventually) be removed.

Reply:
makes sense. We'll do it in a followup PR